testing/fpgaconf/test_fpgaconf_c.cpp

```diff
@@ -728,12 +882,32 @@ TEST_P(fpgaconf_c_p, embed_nullchar2) {
   strcpy(two, "-n");
   strcpy(three, "-B");
   strcpy(four, "0x5e");
-  strcpy(five, "\0copy_bitstream.gbs");
 
   char *argv[] = { zero, one, two, three, four,
                    five };
-  EXPECT_NE(fpgaconf_main(6, argv), 0);
-  unlink(copy_gbs_.c_str());
+
+  // Create link directories
+  if (!(mkdir("./link1", S_IRWXU | S_IRWXG | S_IROTH | S_IXOTH))
```

Comment:
Fail if the mkdir or symlink calls fail?

Reply:
I believe so. But I have yet figure out how to get mkdir and symlink to fail. Any suggestions?